althea_kernel_interface/src/iface_counter.rs

```diff
@@ -23,28 +39,87 @@ impl KernelInterface {
         Ok(())
     }
 
-    /// returns ((input bytes, input packets), (output bytes, output packets))
-    pub fn read_iface_counters(&self, interface: &str) -> Result<((u64, u64), (u64, u64)), Error> {
+    /// returns (input counters, output counters)
+    pub fn read_iface_counters(
+        &self,
+        interface: &str,
+    ) -> Result<(IfaceCounter, IfaceCounter), Error> {
+        lazy_static! {
```

Comment:
this is probably a good habit, likewise I'd like to find some way to hide all 'trace' statements behind a dev build feature, but I'm not sure we can do it for all of our deps too, which is where the big savings would happen. Or maybe the logging crate is a lot more efficient than regex